

float32[] top_left # xmin,ymin

float32[] bottom_right # xmax, ymax 


